.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -105,14 +204,28 @@ jobs:
         with:
           workflow: .github/workflows/rawls-swat-tests.yaml
           repo: broadinstitute/terra-github-workflows
-          ref: refs/heads/main
+          ref: refs/heads/iv-az-e2e-1
+          token: ${{ env.TOKEN }}
+          inputs: '{ "bee-name": "${{ env.BEE_NAME }}", "ENV": "qa", "ref": "refs/heads/${{ needs.init-github-context.outputs.branch }}", "test-group-name": "workspaces_azure", "test-command": "${{ env.rawls_test_command }}", "java-version": "17", "billing-project": "${{ needs.az-e2e-params-gen.outputs.project-name }}", "e2e": "true", "service-account": "${{ inputs.service-account }}" }'
+
+  delete-billing-project-v2-from-bee-workflow:
```

Comment:
If we're going to blow up the bee, do we need to do this step? (could be useful for resetting tests and not creating a bee though)

Reply:
If developers want to do some debugging without recreating BEE, they could use this to remove the billing project and dissociate from the LZ. Otherwise if the BEE will be destroyed, this step is redundant, including it won't have much impact on the test performance.